scripts/docker/dockerfile_gcc-8

```diff
@@ -1,25 +1,38 @@
 FROM axom/compilers:gcc-8
+LABEL maintainer="white238@llnl.gov"
+ARG branch=develop
 
+SHELL ["/bin/bash", "-c"]
 RUN sudo apt-get update -y
-RUN sudo apt-get install curl -fy
-
+RUN sudo apt-get install curl wget ssh pkg-config build-essential libopenblas-dev tar python3 -fy
 RUN sudo apt-get -qq install -y --no-install-recommends gfortran-7 && sudo update-alternatives --install /usr/bin/gfortran gfortran /usr/bin/gfortran-7 100
+RUN sudo ln -f -s python3 /usr/bin/python # Compatibility with legacy Python
 
-RUN git clone --recursive https://github.com/LLNL/axom.git
+# Avoid building CMake from scratch
+RUN wget https://cmake.org/files/v3.10/cmake-3.10.1-Linux-x86_64.sh
+RUN sudo sh cmake-3.10.1-Linux-x86_64.sh --prefix=/usr --exclude-subdir
```

Comment:
Another question -- is there a specific reason to use `cmake-3.10` ? Or is it just that we need a `CMake`, and this version is known to work?

Reply:
known to work, copied from somewhere, not necessary, and should be updated.

Reply:
#404